keyboards/converter/periboard_512/readme.md

```diff
@@ -0,0 +1,47 @@
+# Periboard-512 QMK Conversion
```

Comment:
This file should follow the [template](https://raw.githubusercontent.com/qmk/qmk_firmware/master/quantum/template/avr/readme.md).

Reply:
I will take a look at this soon.